### DISCUSSION TITLE
Admin only django app

### DISCUSSION THREAD

**Discussion by guntribam** [django-commons/django-unicorn#532] [Q&A]
Hey,

i'm starting to build some internal tools with django. I'm only using the generated admin with few customizations. 

Sorry if this is a dumb question, but how can this project fit with an admin-only app?

I'm having some problems that i believe this project would be great:

- dependend combo boxes;
- hidden a field until some value on a preview field is presented;

**Comment by xqzmes** [comment]
Hey!
You can use just the admin panel, but the scope of its application is only the interaction with the database.
Even in this case it is not the best option

**Reply by guntribam** [reply]
After trying a lot of things in django-admin, I had an epiphany and realized that Django admin might be one of the best low-code tools I could use.

Despite this, I still see a lot of bureaucracy to deal with client side javascript, but this is typical of "MPA" frameworks. That's why I thought unicorn could fill this gap in django-admin
